src/App.js

```diff
@@ -1,10 +1,19 @@
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
-import React, { Suspense } from "react";
+import React, { Suspense, useState } from "react";
 import "./App.css";
 import Header from "./components/Header";
 import Home from "./components/Home";
+import { useTranslation } from "react-i18next";
 
 function App() {
+  // const [t, i18n] = useTranslation("common");
```

Comment:
Try not to have commented code in the PR to review.

Reply:
yes, okay